src/components/SciApplicationDetailTemplate.vue

```diff
@@ -42,7 +42,7 @@
             </tr>
             <tr v-for="(timeRequest, index) in timeRequestGroup" :key="semester + '-' + index">
               <td v-if="isKeyApplication"></td>
-              <td>{{ timeRequest.instrument_name }}</td>
+              <td>{{ timeRequest.instrument_names.join() }}</td>
```

Comment:
Does this end up looking like a list of instrument names without spaces after the commas? If so, might be nice to make it so that there are spaces.

Reply:
I'll change it to .join(', ') I guess. In practice it looked okay on my monitors with no space because most of the instrument_types ended up on their own row/line due to the smaller size of the column they are in, but It's probably better to have the space after the comma.